kura/org.eclipse.kura.linux.watchdog/src/main/java/org/eclipse/kura/linux/watchdog/WatchdogServiceImpl.java

```diff
@@ -77,6 +80,31 @@ public void updated(Map<String, Object> properties) {
         });
     }
 
+    private void stopWatchdogd() {
```

Comment:
I don't remember what was exactly the issue and you have probably tried to find the best solution but would it be possible to determine which system is running and execute the corresponding command rather to try all possible commands ?

Reply:
We are trying to step back from this habit, in order to prevent the hardcoding of platforms in Kura code. For now we try to support the maximum. The plan is to be able to define a better solution for the next release, not only for watchdog but also for networking.

Reply:
This is ok for me, no problem